sdk/spring-sdk/src/it/resources/application.conf

```diff
@@ -0,0 +1 @@
+kalix.event-sourced-entity.snapshot-every = 10
```

Comment:
I don't remember the reasons, but I know that it's better to have one empty line at the end of .conf files. I think it's related to merging files. @patriknw can give more insight for sure.

Reply:
added now, tks